R/slab.R

```diff
@@ -27,202 +23,163 @@
 	names(pt) <- make.names(levels(values))
 
 	return(pt)
-	}
-
-
-## TODO: make these exported functions with documentation (https://github.com/ncss-tech/aqp/issues/99)
-
+}
 
+.slab.fun.factor.weighted <- function(values, w, na.rm = TRUE, ...) {
```

Comment:
I'm thinking that weighted / unweighted functions be merged into a single function with a NULL `w` argument. When it is specified the weighted version of e.g. `mean` is used.

Reply:
This probably can be done along with the below comments about `aggregate()` -> data.table